src/components/views/audio_messages/PlayPauseButton.tsx

```diff
@@ -14,28 +14,28 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import React, { ReactNode } from "react";
+import React, { ComponentProps, ReactNode } from "react";
 import classNames from "classnames";
 
 import AccessibleTooltipButton from "../elements/AccessibleTooltipButton";
 import { _t } from "../../../languageHandler";
 import { Playback, PlaybackState } from "../../../audio/Playback";
 
 // omitted props are handled by render function
-interface IProps extends Omit<React.ComponentProps<typeof AccessibleTooltipButton>, "title" | "onClick" | "disabled"> {
+type Props = Omit<ComponentProps<typeof AccessibleTooltipButton>, "title" | "onClick" | "disabled" | "element"> & {
```

Comment:
Is `element` omitted here because a `PlayPauseButton` can only ever be based on the default element type (`div`)?

Not entirely convinced that the comment here is accurate; or indeed giving any value. Maybe remove it?

Reply:
> can only ever be based on the default element type (div)?

Indeed, really I'm not sure this should be extending ComponentProps and instead should just list the ones it wants given it isn't a generic reusable component but I thought I'd apply a consistent approach rather than dig into less relevant things